File: CS/ConsoleApplication1/Quote.cs
using DevExpress.Xpo;

namespace ConsoleApplication1 {

    public class Quote : XPObject {
        public Quote()
            : base() {
            // This constructor is used when an object is loaded from a persistent storage.
            // Do not place any code here.
        }
        public Quote(Session session)
            : base(session) {
            // This constructor is used when an object is loaded from a persistent storage.
            // Do not place any code here.
        }
        public override void AfterConstruction() {
            base.AfterConstruction();
            // Place here your initialization code.
        }
        private string _name;
        public string Name {
            get { return _name; }
            set { SetPropertyValue("Name", ref _name, value); }
        }
        [Association("Quote-QuoteItems"), Aggregated]
        public XPCollection<QuoteItem> QuoteItems {
            get {
                return GetCollection<QuoteItem>("QuoteItems");
            }
        }
    }
}

File: VB/ConsoleApplication1 - VB/Quote.vb
Imports DevExpress.Xpo

Namespace ConsoleApplication1

	Public Class Quote
		Inherits XPObject

		Public Sub New()
			MyBase.New()
			' This constructor is used when an object is loaded from a persistent storage.
			' Do not place any code here.
		End Sub
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
			' This constructor is used when an object is loaded from a persistent storage.
			' Do not place any code here.
		End Sub
		Public Overrides Sub AfterConstruction()
			MyBase.AfterConstruction()
			' Place here your initialization code.
		End Sub
		Private _name As String
		Public Property Name() As String
			Get
				Return _name
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Name", _name, value)
			End Set
		End Property
		<Association("Quote-QuoteItems"), Aggregated>
		Public ReadOnly Property QuoteItems() As XPCollection(Of QuoteItem)
			Get
				Return GetCollection(Of QuoteItem)("QuoteItems")
			End Get
		End Property
	End Class
End Namespace